customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -743,4 +723,70 @@ public String toString() {
             return toJsonString();
         }
     }
+
+    @DynamoDbBean
+    public static class ServiceCenterDao implements JsonSerializable {
+
+        private URI uri;
+        private String text;
```

Comment:
What is "text" here? The display name? Is it possible that this is multilingual?

Reply:
Conventionally, name is the "name", i.e. it is only called this thing…while label can be multilingual